Userland/Applications/PixelPaint/ImageEditor.cpp

```diff
@@ -203,6 +275,11 @@ void ImageEditor::mousedown_event(GUI::MouseEvent& event)
 
 void ImageEditor::mousemove_event(GUI::MouseEvent& event)
 {
+    auto old_mouse_position = m_mouse_position;
+    m_mouse_position = event.position();
+    if (m_mouse_position != old_mouse_position)
+        update(); // TODO: Limit to indicator rects
```

Comment:
Does the `mousemove_event()` ever get called when the cursor position _isn't_ different? I don't think that is the case (if it is, then that feels confusing). The extra check here might not be necessary.

Reply:
Yeah, that logic is unnecessary, will clean that up 👍